connection.hpp

```diff
@@ -69,7 +68,7 @@ class connection
   request request_;
 
   /// The parser for the incoming request.
-  request_parser request_parser_;
+  ///request_parser request_parser_;
```

Comment:
Maybe just get rid of this altogether since we're no longer using it? Unless it could be useful for later parts of the project